packages/rpc/src/common.ts

```diff
@@ -107,7 +107,7 @@ export type RpcRequest = WithHash<{
     ccmMetadata?: Nullish<CcmParams>,
     boostFee?: Nullish<number>,
     affiliateFees?: Nullish<{ account: string; bps: number }[]>,
-    fillOrKillParams?: Nullish<FillOrKillParams>,
+    fillOrKillParams?: FillOrKillParams,
```

Comment:
will be mandatory starting with 1.8: https://github.com/chainflip-io/chainflip-backend/pull/5694/files